docs/lib/predictions/fragments/android/identify-text.md

```diff
@@ -1,3 +1,5 @@
+The following APIs will allow you to identify text (words, tables, pages from a book) from an image.
```

Comment:
I like calling out examples of specific cases that we support because text in an image is really vague.

Words I interpret as words that may not appear in a linear form.. perhaps you can imagine an advertisement with words spaced out in various places with various fonts

Tables I interpret to be words an numbers that often line up on a X or Y coordinate system

Pages from a book i would imagine would be mostly words that line up on vertical points on a page.

I think calling out shows flexibility/versatility of the API, but if you don't think it's adding much value we can remove it.